21:19:15 ConfigMap 08db1feb.percona.com 812:	percona-xtradb-cluster-operator-6db54dc756-fdczk_1ca24eb0-99a9-4932-8a9f-9321d9c82811 became leader
21:19:15 Lease 08db1feb.percona.com 813:	percona-xtradb-cluster-operator-6db54dc756-fdczk_1ca24eb0-99a9-4932-8a9f-9321d9c82811 became leader
21:19:46 ConfigMap 08db1feb.percona.com 910:	percona-xtradb-cluster-operator-64c4fd5974-6s9js_18e6e370-3adf-406c-bbf1-2f5630a5ec1a became leader
21:19:46 Lease 08db1feb.percona.com 911:	percona-xtradb-cluster-operator-64c4fd5974-6s9js_18e6e370-3adf-406c-bbf1-2f5630a5ec1a became leader
21:25:25 PersistentVolumeClaim datadir-test-cluster-pxc-0 1768:	waiting for first consumer to be created before binding
21:25:25 PersistentVolumeClaim datadir-test-cluster-pxc-0 1775:	waiting for a volume to be created, either by external provisioner "rancher.io/local-path" or manually created by system administrator
21:25:25 PersistentVolumeClaim datadir-test-cluster-pxc-0 1775:	External provisioner is provisioning volume for claim "acto-namespace/datadir-test-cluster-pxc-0"
21:25:30 PersistentVolumeClaim datadir-test-cluster-pxc-0 1775:	Successfully provisioned volume pvc-6620b740-2fd8-4141-9dad-543e19ce4269
21:26:31 PersistentVolumeClaim datadir-test-cluster-pxc-1 2068:	waiting for first consumer to be created before binding
21:26:31 PersistentVolumeClaim datadir-test-cluster-pxc-1 2075:	waiting for a volume to be created, either by external provisioner "rancher.io/local-path" or manually created by system administrator
21:26:31 PersistentVolumeClaim datadir-test-cluster-pxc-1 2075:	External provisioner is provisioning volume for claim "acto-namespace/datadir-test-cluster-pxc-1"
21:26:35 PersistentVolumeClaim datadir-test-cluster-pxc-1 2075:	Successfully provisioned volume pvc-4c15eac2-3b40-4e92-820e-060f162fc808
21:27:36 PersistentVolumeClaim datadir-test-cluster-pxc-2 2374:	waiting for first consumer to be created before binding
21:27:36 PersistentVolumeClaim datadir-test-cluster-pxc-2 2381:	waiting for a volume to be created, either by external provisioner "rancher.io/local-path" or manually created by system administrator
21:27:36 PersistentVolumeClaim datadir-test-cluster-pxc-2 2381:	External provisioner is provisioning volume for claim "acto-namespace/datadir-test-cluster-pxc-2"
21:27:40 PersistentVolumeClaim datadir-test-cluster-pxc-2 2381:	Successfully provisioned volume pvc-dfef78a8-c9d9-4d09-8dd3-2418b5ce1fa3
21:19:17 Pod percona-xtradb-cluster-operator-64c4fd5974-6s9js 832:	Successfully assigned acto-namespace/percona-xtradb-cluster-operator-64c4fd5974-6s9js to acto-cluster-6-worker
21:19:19 Pod percona-xtradb-cluster-operator-64c4fd5974-6s9js 834:	Pulling image "percona/percona-xtradb-cluster-operator:1.11.0"
21:19:19 Pod percona-xtradb-cluster-operator-64c4fd5974-6s9js 834:	Successfully pulled image "percona/percona-xtradb-cluster-operator:1.11.0" in 246.689928ms
21:19:19 Pod percona-xtradb-cluster-operator-64c4fd5974-6s9js 834:	Created container percona-xtradb-cluster-operator
21:19:20 Pod percona-xtradb-cluster-operator-64c4fd5974-6s9js 834:	Started container percona-xtradb-cluster-operator
21:19:17 ReplicaSet percona-xtradb-cluster-operator-64c4fd5974 829:	Created pod: percona-xtradb-cluster-operator-64c4fd5974-6s9js
21:19:07 Pod percona-xtradb-cluster-operator-6db54dc756-fdczk 771:	Successfully assigned acto-namespace/percona-xtradb-cluster-operator-6db54dc756-fdczk to acto-cluster-6-worker
21:19:08 Pod percona-xtradb-cluster-operator-6db54dc756-fdczk 774:	Pulling image "percona/percona-xtradb-cluster-operator:1.11.0"
21:19:09 Pod percona-xtradb-cluster-operator-6db54dc756-fdczk 774:	Successfully pulled image "percona/percona-xtradb-cluster-operator:1.11.0" in 585.82549ms
21:19:09 Pod percona-xtradb-cluster-operator-6db54dc756-fdczk 774:	Created container percona-xtradb-cluster-operator
21:19:10 Pod percona-xtradb-cluster-operator-6db54dc756-fdczk 774:	Started container percona-xtradb-cluster-operator
21:19:17 Pod percona-xtradb-cluster-operator-6db54dc756-fdczk 774:	Stopping container percona-xtradb-cluster-operator
21:19:07 ReplicaSet percona-xtradb-cluster-operator-6db54dc756 768:	Created pod: percona-xtradb-cluster-operator-6db54dc756-fdczk
21:19:17 ReplicaSet percona-xtradb-cluster-operator-6db54dc756 836:	Deleted pod: percona-xtradb-cluster-operator-6db54dc756-fdczk
21:19:07 Deployment percona-xtradb-cluster-operator 767:	Scaled up replica set percona-xtradb-cluster-operator-6db54dc756 to 1
21:19:17 Deployment percona-xtradb-cluster-operator 828:	Scaled up replica set percona-xtradb-cluster-operator-64c4fd5974 to 1
21:19:17 Deployment percona-xtradb-cluster-operator 828:	Scaled down replica set percona-xtradb-cluster-operator-6db54dc756 to 0
21:25:25 Pod test-cluster-haproxy-0 1799:	Successfully assigned acto-namespace/test-cluster-haproxy-0 to acto-cluster-6-worker3
21:25:26 Pod test-cluster-haproxy-0 1805:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
21:25:26 Pod test-cluster-haproxy-0 1805:	Created container haproxy
21:25:26 Pod test-cluster-haproxy-0 1805:	Started container haproxy
21:25:26 Pod test-cluster-haproxy-0 1805:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
21:25:26 Pod test-cluster-haproxy-0 1805:	Created container pxc-monit
21:25:26 Pod test-cluster-haproxy-0 1805:	Started container pxc-monit
21:25:45 Pod test-cluster-haproxy-0 1805:	Readiness probe failed: ERROR 2013 (HY000): Lost connection to MySQL server at 'reading initial communication packet', system error: 2

21:26:35 Pod test-cluster-haproxy-1 2121:	Successfully assigned acto-namespace/test-cluster-haproxy-1 to acto-cluster-6-worker2
21:26:36 Pod test-cluster-haproxy-1 2123:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
21:26:36 Pod test-cluster-haproxy-1 2123:	Created container haproxy
21:26:36 Pod test-cluster-haproxy-1 2123:	Started container haproxy
21:26:36 Pod test-cluster-haproxy-1 2123:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
21:26:36 Pod test-cluster-haproxy-1 2123:	Created container pxc-monit
21:26:37 Pod test-cluster-haproxy-1 2123:	Started container pxc-monit
21:26:56 Pod test-cluster-haproxy-2 2223:	Successfully assigned acto-namespace/test-cluster-haproxy-2 to acto-cluster-6-worker
21:26:56 Pod test-cluster-haproxy-2 2225:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
21:26:56 Pod test-cluster-haproxy-2 2225:	Created container haproxy
21:26:56 Pod test-cluster-haproxy-2 2225:	Started container haproxy
21:26:56 Pod test-cluster-haproxy-2 2225:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
21:26:56 Pod test-cluster-haproxy-2 2225:	Created container pxc-monit
21:26:57 Pod test-cluster-haproxy-2 2225:	Started container pxc-monit
21:33:18 Pod test-cluster-haproxy-2 2225:	Stopping container haproxy
21:33:18 Pod test-cluster-haproxy-2 2225:	Stopping container pxc-monit
21:43:20 Pod test-cluster-haproxy-2 5116:	Successfully assigned acto-namespace/test-cluster-haproxy-2 to acto-cluster-6-worker
21:43:21 Pod test-cluster-haproxy-2 5117:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
21:43:21 Pod test-cluster-haproxy-2 5117:	Created container haproxy
21:43:21 Pod test-cluster-haproxy-2 5117:	Started container haproxy
21:43:21 Pod test-cluster-haproxy-2 5117:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
21:43:21 Pod test-cluster-haproxy-2 5117:	Created container pxc-monit
21:43:21 Pod test-cluster-haproxy-2 5117:	Started container pxc-monit
21:25:25 PodDisruptionBudget test-cluster-haproxy 1796:	No matching pods found
21:25:25 StatefulSet test-cluster-haproxy 1787:	create Pod test-cluster-haproxy-0 in StatefulSet test-cluster-haproxy successful
21:26:35 StatefulSet test-cluster-haproxy 1828:	create Pod test-cluster-haproxy-1 in StatefulSet test-cluster-haproxy successful
21:26:56 StatefulSet test-cluster-haproxy 2124:	create Pod test-cluster-haproxy-2 in StatefulSet test-cluster-haproxy successful
21:33:18 StatefulSet test-cluster-haproxy 3390:	delete Pod test-cluster-haproxy-2 in StatefulSet test-cluster-haproxy successful
21:33:19 StatefulSet test-cluster-haproxy 3408:	create Pod test-cluster-haproxy-2 in StatefulSet test-cluster-haproxy failed error: Pod "test-cluster-haproxy-2" is invalid: spec.containers[1].resources.requests: Invalid value: "800m": must be less than or equal to cpu limit
21:25:31 Pod test-cluster-pxc-0 1770:	Successfully assigned acto-namespace/test-cluster-pxc-0 to acto-cluster-6-worker2
21:25:32 Pod test-cluster-pxc-0 1861:	Container image "percona/percona-xtradb-cluster-operator:1.11.0" already present on machine
21:25:32 Pod test-cluster-pxc-0 1861:	Created container pxc-init
21:25:32 Pod test-cluster-pxc-0 1861:	Started container pxc-init
21:25:33 Pod test-cluster-pxc-0 1861:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
21:25:33 Pod test-cluster-pxc-0 1861:	Created container logs
21:25:33 Pod test-cluster-pxc-0 1861:	Started container logs
21:25:33 Pod test-cluster-pxc-0 1861:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
21:25:33 Pod test-cluster-pxc-0 1861:	Created container logrotate
21:25:34 Pod test-cluster-pxc-0 1861:	Started container logrotate
21:25:34 Pod test-cluster-pxc-0 1861:	Container image "percona/percona-xtradb-cluster:8.0.27-18.1" already present on machine
21:25:34 Pod test-cluster-pxc-0 1861:	Created container pxc
21:25:34 Pod test-cluster-pxc-0 1861:	Started container pxc
21:26:01 Pod test-cluster-pxc-0 1861:	Readiness probe failed: ERROR 2003 (HY000): Can't connect to MySQL server on '10.244.3.3:33062' (111)
+ [[ '' == \P\r\i\m\a\r\y ]]
+ exit 1

21:26:36 Pod test-cluster-pxc-1 2071:	Successfully assigned acto-namespace/test-cluster-pxc-1 to acto-cluster-6-worker
21:26:37 Pod test-cluster-pxc-1 2133:	Container image "percona/percona-xtradb-cluster-operator:1.11.0" already present on machine
21:26:37 Pod test-cluster-pxc-1 2133:	Created container pxc-init
21:26:38 Pod test-cluster-pxc-1 2133:	Started container pxc-init
21:26:38 Pod test-cluster-pxc-1 2133:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
21:26:38 Pod test-cluster-pxc-1 2133:	Created container logs
21:26:39 Pod test-cluster-pxc-1 2133:	Started container logs
21:26:39 Pod test-cluster-pxc-1 2133:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
21:26:39 Pod test-cluster-pxc-1 2133:	Created container logrotate
21:26:39 Pod test-cluster-pxc-1 2133:	Started container logrotate
21:26:39 Pod test-cluster-pxc-1 2133:	Container image "percona/percona-xtradb-cluster:8.0.27-18.1" already present on machine
21:26:39 Pod test-cluster-pxc-1 2133:	Created container pxc
21:26:39 Pod test-cluster-pxc-1 2133:	Started container pxc
21:27:06 Pod test-cluster-pxc-1 2133:	Readiness probe failed: ERROR 2003 (HY000): Can't connect to MySQL server on '10.244.2.5:33062' (111)
+ [[ '' == \P\r\i\m\a\r\y ]]
+ exit 1

21:27:40 Pod test-cluster-pxc-2 2377:	Successfully assigned acto-namespace/test-cluster-pxc-2 to acto-cluster-6-worker3
21:27:41 Pod test-cluster-pxc-2 2411:	Container image "percona/percona-xtradb-cluster-operator:1.11.0" already present on machine
21:27:41 Pod test-cluster-pxc-2 2411:	Created container pxc-init
21:27:42 Pod test-cluster-pxc-2 2411:	Started container pxc-init
21:27:42 Pod test-cluster-pxc-2 2411:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
21:27:42 Pod test-cluster-pxc-2 2411:	Created container logs
21:27:42 Pod test-cluster-pxc-2 2411:	Started container logs
21:27:42 Pod test-cluster-pxc-2 2411:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
21:27:42 Pod test-cluster-pxc-2 2411:	Created container logrotate
21:27:43 Pod test-cluster-pxc-2 2411:	Started container logrotate
21:27:43 Pod test-cluster-pxc-2 2411:	Container image "percona/percona-xtradb-cluster:8.0.27-18.1" already present on machine
21:27:43 Pod test-cluster-pxc-2 2411:	Created container pxc
21:27:43 Pod test-cluster-pxc-2 2411:	Started container pxc
21:28:11 Pod test-cluster-pxc-2 2411:	Readiness probe failed: ERROR 2003 (HY000): Can't connect to MySQL server on '10.244.1.4:33062' (111)
+ [[ '' == \P\r\i\m\a\r\y ]]
+ exit 1

21:25:25 StatefulSet test-cluster-pxc 1766:	create Claim datadir-test-cluster-pxc-0 Pod test-cluster-pxc-0 in StatefulSet test-cluster-pxc success
21:25:25 StatefulSet test-cluster-pxc 1766:	create Pod test-cluster-pxc-0 in StatefulSet test-cluster-pxc successful
21:26:31 StatefulSet test-cluster-pxc 2054:	create Claim datadir-test-cluster-pxc-1 Pod test-cluster-pxc-1 in StatefulSet test-cluster-pxc success
21:26:31 StatefulSet test-cluster-pxc 2054:	create Pod test-cluster-pxc-1 in StatefulSet test-cluster-pxc successful
21:27:36 StatefulSet test-cluster-pxc 2367:	create Claim datadir-test-cluster-pxc-2 Pod test-cluster-pxc-2 in StatefulSet test-cluster-pxc success
21:27:36 StatefulSet test-cluster-pxc 2367:	create Pod test-cluster-pxc-2 in StatefulSet test-cluster-pxc successful
